dataprep/eda/correlation/compute.py

```diff
@@ -146,17 +148,8 @@ def compute_correlation(
         raise ValueError("Please give the value to x instead of y")
     elif x is not None and y is not None:
         assert value_range is None
-        assert x in df.columns, f"{x} not in columns names"
-        assert y in df.columns, f"{y} not in columns names"
```

Comment:
Why remove the column existence check?

Reply:
Sorry, it is a mistake. I have fixed this problem. Thanks!

Reply:
There are more unexpected deletions of existence check I didn't give a comment. Please fix them, thanks!

Reply:
According to #48, in order to output empty figures, I have deleted several checks to avoid raising errors. I will double-check the deletions. Thanks!